src/js/index.js

```diff
@@ -16,6 +18,25 @@ var reparse = require('./actions/vegaInvalidate');
 var model = require('./model');
 model.init();
 
+// Set up the listeners that connect the model to the store
+var listeners = require('./store/listeners');
+
+// Bind the listener that will flow changes from the redux store into Vega
+store.subscribe(listeners.createStoreListener(
+  store,
+  model,
+  listeners.recreateVegaIfNecessary,
```

Comment:
They're just functions, and absolutely could be referenced from inside the file -- doing things this way is unorthodox but is intended to make it easier to test these listener methods, since we can pass in a mock store and model with spies to validate the branching. Those tests are yet to be written but my notion with the file was, all non-utility dependencies should be explicitly injected as arguments rather than relying on implicit relationships.